app/xray/config.py

```diff
@@ -302,7 +302,7 @@ def include_db_users(self) -> XRayConfig:
                 for proxy_type, inbound_tags in user.inbounds.items():
                     for inbound_tag in inbound_tags:
                         config.add_inbound_client(inbound_tag,
-                                                  user.username,
+                                                  f"{user.id}.{user.username}",
```

Comment:
I like your approach. however, we've to modify the xray.operations too. I'll do it

Reply:
ok, got it.

i add subscription feature, it will add a lot of codes https://github.com/Gozargah/Marzban/issues/273#issuecomment-1558376979.  because that depend this patch, so if this patch can merged, i can push next request easyly.